workspaces/global-floating-action-button/plugins/global-floating-action-button/src/types.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum Position {
+  bottomRight = 'bottomRight',
+  bottomCenter = 'bottomCenter',
+}
```

Comment:
I know that's now really opinionated, and I don't know if we have any rule for that, so I checked in backstage code and found less enums then expected. But at least this two:

https://github.com/backstage/backstage/blob/6b92433d5e68bc57174ed9c6546d085639f82dee/plugins/scaffolder-backend-module-gitlab/report.api.md?plain=1#L266-L284

https://github.com/backstage/backstage/blob/6b92433d5e68bc57174ed9c6546d085639f82dee/plugins/catalog-graph/src/components/EntityRelationsGraph/types.ts#L110-L127

So I would suggest at least uppercase KEYS and - or _ as delimiter for the values. I'm thinking how to define that in YAML later.

```suggestion
export enum Position {
  BOTTOM_RIGHT = 'bottom-right',
  BOTTOM_CENTER = 'bottom-center',
}
```

Btw. I would call slot and "page end instead of right to support (somewhere in the future) RTL.

```suggestion
export enum Slot {
  PAGE_END = 'page-end',
  BOTTOM_CENTER = 'bottom-center',
}
```

But that's all just loud thinking. :D